devices/Tm1637/README.md

```diff
@@ -18,7 +20,6 @@ You need to create a ```Tm1637``` class with 2 pins, the clock one and the data
 Tm1637 tm1637 = new Tm1637(21, 20);
 ```
 
-![tm1637](./tm1637_bb.png)
```

Comment:
I've been recently remaking all our fritzing diagrams to include an ESP32. Can You do the same for this device? That would be super nice.